recipes/maxit/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "MAXIT" %}
+{% set version = "11.200" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://sw-tools.rcsb.org/apps/{{ name }}/{{ name|lower }}-v{{ version }}-prod-src.tar.gz
+  sha256: 658e236c6310cf7e55218a0500f82050ef86074f5d5b3f61d2a161b04a38cc39
+  patches:
+    - platform.sh.patch  # For osx-arm64
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}  # [linux]
```

Comment:
If there is no CXX for OSX then https://github.com/bioconda/bioconda-recipes/pull/50461/files#diff-0a3873f67428cfad405c42d1950c9702d2031f4d1c222c30d1e2f3e8a86ed7e2R14 would fail, no ?

Reply:
@martin-g 
Basically, the build will fail if there is no CXX for OSX.

However, it may succeed if any of GCC v2-4 is installed on MacOS build environment, which is supposed to be not possible today.

Reply:
@eunos-1128 Without `- {{ compiler('cxx') }}` there is no `$CXX`, so on OSX the symlink will have no source

Reply:
On MacOS, clangxx will be installed even without `- {{ compiler('cxx') }}`, which means `- {{ compiler('cxx') }}` notation is necessary only for linux.

Reply:
> clangxx will be installed even without `- {{ compiler('cxx') }}`, which means `- {{ compiler('cxx') }}` notation is necessary only for linux

This sounds like a bug/coincidence to me.

Reply:
Yeah, I found it is bug, too.

Reply:
@martin-g 

Changing the line from `- {{ compiler('cxx') }}  # [linux]` to `- {{ compiler('cxx') }}` make difference to build results, in both of which build succeeded.  

https://github.com/bioconda/bioconda-recipes/pull/50461/files#diff-52281e768243184b4edcff9c517394ef6046c4d0e16500ed9b2d245b7bb621b2R22

Reply:
This was because you were using `$BUILD_PREFIX/bin/clang++` instead of  the recommended `$CXX`